src/main/webapp/assets/modules/common/validator.ser.js

```diff
@@ -0,0 +1,15 @@
+angular.module("Common").factory("validatorService", function ($translate) {
+
+  var instance = {
+    validate: function(elems) {
+      var error = {};
+      $.each(elems, function(i, elem) {
+        $(elem).hasClass("ng-invalid-required") && (error[$(elem).attr("ng-model")] = $translate.instant('requiredThisField'));
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Expected an assignment or function call and instead saw an expression.